pkg/features/features.go

```diff
@@ -41,4 +47,5 @@ const (
 var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	RequireAuthorization: {Default: false, PreRelease: featuregate.Alpha},
 	ModuleRestart:        {Default: false, PreRelease: featuregate.Alpha},
+	DisableCSI:           {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
the variable name should be the meaning of the flag without boolean? having disable in the name could be complicated for user to set.

```suggestion
	SupportCSINode:           {Default: true, PreRelease: featuregate.Alpha},
```